### PR TITLE
added Mohit Gadireddy

### LINKED ISSUE #5051: [Community] Member Profile: Mohith Gadireddy
**Current Behavior**

@Mohith234 has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @Mohith234 as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @Mohith234
- Twitter: 
- LinkedIn: [in/mohith-gadireddy](https://linkedin.com/in/mohith-gadireddy)
- Meshery Cloud: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Harsha-10**
**Description**

This PR fixes #5051

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 671320233cf4eefa37e705fa4837bc30656f2410 at: https://652fc25b1989fd18fb00db09--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 961ddc8887f22dd4ab9db51c7dcd6b2939d65d27 at: https://652fc981f4e7dd23e92d3e8f--layer5.netlify.app

**Comment by Harsha-10** [comment]
please review @Mohith234 @saurabh100ni

**Comment by Mohith234** [comment]
@Harsha-10 Could you replace the image with [this](https://drive.google.com/file/d/10o-Y09LWcHYz9ff_6KAxbzOloiSda294/view?usp=sharing)

**Comment by Harsha-10** [comment]
Okay

**Comment by Harsha-10** [comment]
image

**Comment by l5io** [comment]
🚀 Preview for commit 8795bd2f2e018d6ea91948225b082d9a82f56bf3 at: https://652feaab3a2c075171815794--layer5.netlify.app

File: src/collections/members/Mohith-Gadireddy/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Mohith Gadireddy
+position: Contributor
+image_path: ./Mohith-Gadireddy.webp
+github: Mohith234
+twitter: mohith_twt
+linkedin: mohith-gadireddy
+layer5: 92b8fb72-fa12-4a7e-9a40-c7f79b292248
+location: Hyderabad , Telangana
```
Comment (Mohith234) [inline src/collections/members/Mohith-Gadireddy/index.mdx:9]: - Just a small suggestion and the rest everything is good
```suggestion
location: Hyderabad, India
```

**Comment by l5io** [comment]
🚀 Preview for commit 4d7b6cc0fb195ca3e0ef0e848d31e00331788f50 at: https://652ffa97fe634b0b69909b13--layer5.netlify.app

**Review by Mohith234 — APPROVED** [review]
LGTM 🚀

Thanks for contributing @Harsha-10 🙌

**Comment by abhijeetgauravm** [comment]
> LGTM 🚀
> 
> Thanks for contributing @Harsha-10 🙌

// @ritiksaxena124 @sudhanshutech

**Review by ritiksaxena124 — APPROVED** [review]

**Pull request merged**